spec/unit/counter_spec.rb

```diff
@@ -1,18 +1,37 @@
 require 'counter'
 
 describe Counter do
-  describe '#new' do
-    it 'starts with a count of 0' do
-      counter = Counter.new
-      expect(counter.count).to eq 0
-    end
-  end
+  before (:each) { @counter = Counter.new }
 
   describe '#increment' do
     it 'adds 1 to the count' do
-      counter = Counter.new
-      counter.increment
-      expect(counter.count).to eq 1
+      @counter.increment
+      expect(@counter.count).to eq 1
     end
   end
+
+  describe '#decrement' do
+    it 'takes 1 from the count' do
+      @counter.decrement
+      expect(@counter.count).to eq -1
+    end
+  end
+
```

Comment:
This is sort of implicitly tested in your #increment and #decrement tests but to be super explicit it would be good to have a test to check that the counter is 0 when the object is first created.

Reply:
Or you could consider using the rspec `change` matcher, which can check for this at the same time as checking that increment/decrement works: https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/change-matcher